tests/chart_tests/test_resources.py

```diff
@@ -0,0 +1,44 @@
+import jmespath
```

Comment:
Can we rename this file to `test_container_resources.py` since the word "resources" is used in other contexts even within kubernetes manifests, for instance within roles. https://github.com/astronomer/astronomer/blob/59ad90435a/charts/astronomer/templates/commander/commander-role.yaml#L41